specification/servicenetworking/ServiceNetworking.Management/main.tsp

```diff
@@ -31,7 +31,7 @@ model Frontend is TrackedResource<FrontendProperties>{
   @visibility("read")
   @path
   @segment("frontends")
-  @pattern("[A-Za-z0-9]+[A-Za-z0-9-_.]{0,62}[A-Za-z0-9_]+")
```

Comment:
I believe this is already updated in main from https://github.com/Azure/azure-rest-api-specs/pull/24225.  Maybe you need to merge latest main to your branch?

Reply:
This is fixed now that the PR is targeting typespec-next